src/main/java/com/stripe/model/Charge.java

```diff
@@ -194,6 +194,14 @@
   @SerializedName("payment_intent")
   String paymentIntent;
 
+  /** ID of the payment method used in this charge. */
+  @SerializedName("payment_method")
+  String paymentMethod;
+
+  /** Transaction-specific details of the payment method used in the payment. */
+  @SerializedName("payment_method_details")
+  PaymentMethodDetails paymentMethodDetails;
+
```

Comment:
We're missing `billing_details` which is also shipping at the same time as `payment_method_details`. I realize it's also missing in my PRs though and I'll have to fix this but I think it's important to fix in pay-server